tools/hive/kerberos/scripts/grant-hive-privileges.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -x
+
+export HADOOP_HOME=/usr/local/hadoop
+
+# Create a user "alice" since Sentry authorization relies on the Linux user and group information
+/usr/sbin/useradd alice
+
+# Grant permissions to user “alice”
+echo "Grant permissions to user alice ..."
+kdestroy
+kinit -k -t /usr/local/hadoop/etc/hadoop/hdfs.keytab hive/${HOSTNAME}@LOCAL
```

Comment:
Does this **have** to be the `hive` user? could one use the `hdfs` user here too?